### ISSUE TITLE
Segmentation fault: 11 MacOS 10.13.2 (homebrew installed bottle)

### ISSUE THREAD

**Issue by Daemeron** [facebook/hhvm#8083]
### HHVM Version
HipHop VM 3.23.2-2brew (rel)
Compiler: 1513100964_N
Repo schema: 05b63be1e86841dd18c1ab071f80c9b5ede8fc18
### Operating System and Version
MacOS 10.13.2 (17C88)
### Standalone code, or other way to reproduce the problem
laravel 4.2 with mysql connection
### Expected result
laravel artisan tinker command line
### Actual result
```
Core dumped: Segmentation fault: 11
Stack trace in /tmp/stacktrace.74460.log
[1]    74460 segmentation fault  hhvm artisan tinker

cat /tmp/stacktrace.74460.log

Host: Dev
ProcessID: 74460
ThreadID: 140735818842944
ThreadPID: -1
Name: /usr/local/bin/hhvm
Type: Segmentation fault: 11
Runtime: hhvm
Version: 1513100964_N
DebuggerCount: 0

Arguments: artisan tinker
ThreadType: CLI


PHP Stacktrace:

#0  PDOStatement->execute() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php:37]
#1  Illuminate\Database\Connectors\MySqlConnector->connect() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Connectors/ConnectionFactory.php:57]
#2  Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Connectors/ConnectionFactory.php:46]
#3  Illuminate\Database\Connectors\ConnectionFactory->make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/DatabaseManager.php:176]
#4  Illuminate\Database\DatabaseManager->makeConnection() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/DatabaseManager.php:64]
#5  Illuminate\Database\DatabaseManager->connection() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:2922]
#6  Illuminate\Database\Eloquent\Model::resolveConnection() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:2888]
#7  Illuminate\Database\Eloquent\Model->getConnection() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1849]
#8  Illuminate\Database\Eloquent\Model->newBaseQueryBuilder() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1791]
#9  Illuminate\Database\Eloquent\Model->newQueryWithoutScopes() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1764]
#10 Illuminate\Database\Eloquent\Model->newQuery() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:3109]
#11 Illuminate\Database\Eloquent\Model->__call()
#12 call_user_func_array() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:3125]
#13 Illuminate\Database\Eloquent\Model::__callStatic() called at [/Users/daemeron/Source/websiteng/src/Tado/Models/ApplicationGlobal.php:112]
#14 Tado\Models\ApplicationGlobal::getApplicationGlobal() called at [/Users/daemeron/Source/websiteng/workbench/tado/webshop/src/Tado/Webshop/Services/Netsuite/NetsuiteOrderSubmissionData.php:94]
#15 Tado\Webshop\Services\Netsuite\NetsuiteOrderSubmissionData->__construct()
#16 ReflectionClass->newInstanceArgs() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:539]
#17 Illuminate\Container\Container->build() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:428]
#18 Illuminate\Container\Container->make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:462]
#19 Illuminate\Foundation\Application->make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:609]
#20 Illuminate\Container\Container->resolveClass() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:570]
#21 Illuminate\Container\Container->getDependencies() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:537]
#22 Illuminate\Container\Container->build() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:428]
#23 Illuminate\Container\Container->make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:462]
#24 Illuminate\Foundation\Application->make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:609]
#25 Illuminate\Container\Container->resolveClass() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:570]
#26 Illuminate\Container\Container->getDependencies() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:537]
#27 Illuminate\Container\Container->build() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Container/Container.php:428]
#28 Illuminate\Container\Container->make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:462]
#29 Illuminate\Foundation\Application->make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Support/helpers.php:33]
#30 app() called at [/Users/daemeron/Source/websiteng/workbench/tado/webshop/src/Tado/Webshop/WebshopServiceProvider.php:156]
#31 Tado\Webshop\WebshopServiceProvider->registerEventSubscribers() called at [/Users/daemeron/Source/websiteng/workbench/tado/webshop/src/Tado/Webshop/WebshopServiceProvider.php:86]
#32 Tado\Webshop\WebshopServiceProvider->boot() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:585]
#33 Closure$Illuminate\Foundation\Application::boot()
#34 array_walk() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:585]
#35 Illuminate\Foundation\Application->boot() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Console/Application.php:44]
#36 Illuminate\Console\Application::make() called at [/Users/daemeron/Source/websiteng/vendor/laravel/framework/src/Illuminate/Console/Application.php:33]
#37 Illuminate\Console\Application::start() called at [/Users/daemeron/Source/websiteng/artisan:46]
```

**Comment by desbo** [comment]
related to (or possibly clone of) #8050

**Comment by fredemmott** [comment]
Yep, closing as dupe. a native stack trace (from LLDB) would be needed to be sure it's not, but it's extremely likely it is. Another possibility was fixed in 3.23.3
